server/helpers/custom-validators/activitypub/videos.ts

```diff
@@ -67,6 +67,7 @@ function sanitizeAndCheckVideoTorrentObject (video: any) {
     isVideoViewsValid(video.views) &&
     isBooleanValid(video.sensitive) &&
     isBooleanValid(video.commentsEnabled) &&
+    isBooleanValid(video.downloadingEnabled) &&
```

Comment:
Add `if (!isBooleanValid(video.downloadingEnabled)) video.downloadingEnabled = true` for backward compatibility